epoxy-processor/src/main/java/com/airbnb/epoxy/ClassToGenerateInfo.java

```diff
@@ -66,7 +84,7 @@ public ClassToGenerateInfo(TypeElement originalClassName, ClassName generatedCla
     this.isOriginalClassAbstract = isOriginalClassAbstract;
   }
 
-  private List<ParameterSpec> buildConstructorParamList(List<? extends TypeMirror> params) {
+  private List<ParameterSpec> buildParamList(List<? extends TypeMirror> params) {
```

Comment:
The only idea that comes to my mind is to have a boolean flag to be able to distinguish between method call for constructor and method, but it looks kinda ugly.